projects/tinkerbell/tinkerbell-chart/chart/values.yaml

```diff
@@ -8,7 +8,7 @@ createNamespace: true
 boots:
   deploy: true
   name: boots
-  image: public.ecr.aws/eks-anywhere/tinkerbell/boots:94e4b4899b383e28b6002750b14e254cfbbdd81f-eks-a-12
+  image: quay.io/tinkerbell/boots:v0.8.0
```

Comment:
Are we going to use images directly from quay.io instead of pushing it to our public ecr and use it from there?

Reply:
No, all images get overwritten in our application code. These image names are rather meaningless.